src/react-infinite.jsx

```diff
@@ -1,6 +1,8 @@
 /* @flow */
 
 var React = global.React || require('react');
+var PropTypes = require('prop-types');
```

Comment:
Thanks for the review @garetht.  I've made this change, but I'm having trouble getting the tests to pass due to the new peerDependencies.  I've added them to the npm install line in travis/test.sh, but this did not appear to have an effect on the Travis CI tests, which fail because prop-types and create-react-class are not being installed.  Any ideas here?